docs/source/guide/lre-1-intro.md

```diff
@@ -0,0 +1,185 @@
+---
+jupytext:
+ text_representation:
+   extension: .md
+   format_name: myst
+   format_version: 0.13
+   jupytext_version: 1.11.1
+kernelspec:
+ display_name: Python 3
+ language: python
+ name: python3
+---
+
+
+# How do I use LRE?
+
+
+LRE works in two steps: generate noise-scaled circuits and apply inference to results from executed circuits.
```

Comment:
Without additional comments on the layerwise aspect, this description could just as well apply to ZNE. While this description is not incorrect, it may be misleading as it could be applied to ZNE or LRE without further discussion on how the noise-scaling on different layers comes into the picture.